core/common/src/main/java/alluxio/security/LoginUser.java

```diff
@@ -97,7 +97,7 @@ private static User login(Configuration conf) throws IOException {
       }
       return userSet.iterator().next();
     } catch (LoginException e) {
-      throw new IOException("Failed to login", e);
+      throw new IOException("Failed to login" + e);
```

Comment:
Does `LoginException` not end up as the cause of the `IOException`? Also, if you want to concat the messages, a space should be added.